test/sdf/unrecognized_elements_with_namespace.sdf

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" ?>
+<sdf version="1.8" xmlns:third_party_software="custom_ns_uri">
+  <model name="joint_incorrect_tags" some_attribute="some value">
```

Comment:
nit: `joint_incorrect_tags` makes it sound this is is not a valid file.

Reply:
Changed the name to `namespaced_tags`